task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -264,6 +265,9 @@
       },
     },
     watch: {
+      task(){
+          this.showFooter = typeof this.task !== 'undefined' && this.task.id !== null && this.task.id !== '' ? false : true;
```

Comment:
Improve this by looking at this link https://developer.mozilla.org/en-US/docs/Glossary/Falsy